tests/lib/rules/one-var.js

```diff
@@ -581,6 +646,7 @@ ruleTester.run("one-var", rule, {
         },
         {
             code: "var a = 1; var b = 2; var x, y; for (var z of foo) {var c = 3, baz = z; for (var d in baz) {}}",
+            output: "var a = 1; var b = 2; var x, y; for (var z of foo) {var c = 3; var baz = z; for (var d in baz) {}}",
```

Comment:
Can you add a test where there is a comment between the variable declarators?

```js
var foo = 1, // comment
    bar = 2;
```

I would expect the comment to be preserved and appear somewhere in the output.

Reply:
Here’s what I have now:

```js
var foo = 1;
var // comment
    bar = 2;
```

Reply:
Do you think it would be possible to change that? I think putting `var` and `bar` on separate lines would be surprising to users.